crates/polars-mem-engine/src/executors/scan/parquet.rs

```diff
@@ -210,20 +209,6 @@ impl ParquetExec {
                                 )?
                                 .finish()?;
 
-                            if let Some(col) = &self.file_options.include_file_paths {
```

Comment:
This was duplicate from a very old implementation approach that didn't push `include_file_paths` into the actual reader.